index.html

```diff
@@ -209,15 +210,14 @@ <h2>Multi-Post <br> Stories</h2>
             </div>
         </section>
         <footer>
-            
             <section id="contact-form">
                 <h4>Contact me</h4>
                 <p>
                 If you have an application you are interested in developing,
                 a feature that you need built or a project that needs coding.
                 I’d love to help with it
                 </p>
-                <form class="input" action="https://formspree.io/f/xlezvgbv" method="post">
+                <form class="input" action="#">
                    <input type="text" class="input-item" name="Name" placeholder="full name" maxlength="30" required>
```

Comment:
- [ ]  Your form action should be configured correctly so that when you submit the information, it should be sent successfully. 
 Action should be similar to  `https://formspree.io/f/{form_id}`
Method value equals `Post`